app/scripts/controllers/permissions/flask/snap-permissions.js

```diff
@@ -15,7 +15,7 @@ import {
 export const buildSnapEndowmentSpecifications = () =>
   Object.values(endowmentPermissionBuilders).reduce(
     (allSpecifications, { targetKey, specificationBuilder }) => {
-      if (!ExcludedSnapEndowments.has(targetKey)) {
+      if (!Object.keys(ExcludedSnapEndowments).includes(targetKey)) {
```

Comment:
Why the change to Object.keys?

Reply:
Because this variable is now an object in the form of `{ permission: errorMessage }` (previously a Set)

Reply:
ah I missed that it was changed, ok cool!